docs/current/guides/snippets/use-secrets/sdk/main.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+import anyio
+import dagger
+
+async def main():
+    async with dagger.Connection(dagger.Config(log_output=sys.stderr)) as client:
+        # set secret as string value
+        secret = client.set_secret("password", "DOCKER-HUB-PASSWORD")
+
+        # create container
+        c = (
```

Comment:
Nit: single character (`c`) is only convention in Go. Suggestion: `ctr = (`.

Reply:
Done